Source/Diagnostics/FlushFormats/FlushFormatCheckpoint.H

```diff
@@ -0,0 +1,23 @@
+#ifndef WARPX_FLUSHFORMATCHECKPOINT_H_
+#define WARPX_FLUSHFORMATCHECKPOINT_H_
+
+#include "FlushFormatPlotfile.H"
+
+class FlushFormatCheckpoint : public FlushFormatPlotfile
```

Comment:
Just saw it removed below :D
```suggestion
class FlushFormatCheckpoint final : public FlushFormatPlotfile
```